src/speed.c

```diff
@@ -194,72 +122,128 @@ void set_max_end_linear_speed(float value)
 }
 
 /**
- * @brief Set speed parameters with default values from a predefined mode.
+ * @brief Speed turn at a defined speed level.
```

Comment:
I think we should deal with this when we re-think the system architecture and source code files organization.

Reply:
I was wrong because I thought that the module speed only set and get speeds and accelerations but it also moved the robot. Ok with deal with this when re-think system.